app/tasks.py

```diff
@@ -74,8 +75,7 @@ def get(self):
                     next_cursor = query.cursor()
                     was_expired = person.is_expired
                     person.put_expiry_flags()
-                    if (utils.get_utcnow() - person.get_effective_expiry_date()
-                        > EXPIRED_TTL):
+                    if (utils.get_utcnow() - person.get_effective_expiry_date() > EXPIRED_TTL):
```

Comment:
One line should be less than 80 characters:
https://google.github.io/styleguide/pyguide.html#Line_length

So you should keep the original format.